src/v/config/configuration.cc

```diff
@@ -1532,6 +1532,12 @@ configuration::configuration()
       "health_monitor_append_entries_timeout_ms",
       "Max time to waiting append_entries request before be isoalted",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      3000)
+  , out_of_raft_heartbeats_timeout_ms(
```

Comment:
"out of raft heartbeat" is probably too implementation-specific for exposing to the user in a config property name.  We haven't really had to invent a polite name for this kind of heartbeat so far, but maybe "node heartbeats" is enough to distinguish them from "raft heartbeats"

if these two properties need to be separate (do they?) then maybe something like:
- `node_isolation_raft_timeout`
- `node_isolation_heartbeat_timeout`